index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Adrian and Faruk's Portfolio</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+ 
+
+ 
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <nav id="navbar">
+        <ul class="nav-list">
+            <li>
+                <a href="#welcome-section">About us</a>
+            </li>
+            <li> 
+                <a href="#projects">Our Projects</a>
+            </li>
+        </ul>
+      </nav>
+    <section
+id="welcome-section" 
+class="welcome-section">
```

Comment:
The id's and classes should be placed inline so code is easier to read and to be on point with the Coding Standards. 😄